src/main/java/eu/dissco/backend/service/SpecimenService.java

```diff
@@ -64,15 +64,16 @@ public class SpecimenService {
   private final AnnotationService annotationService;
   private final MongoRepository mongoRepository;
 
-  public JsonApiListResponseWrapper getSpecimen(int pageNumber, int pageSize, String path) {
-    var digitalSpecimenList = repository.getSpecimensLatest(pageNumber, pageSize);
-    return wrapListResponse(digitalSpecimenList, pageSize, pageNumber, path);
+  public JsonApiListResponseWrapper getSpecimen(int pageNumber, int pageSize, String path)
```

Comment:
Wondering, are the db methods still used?

Reply:
Nope. Thought they were, but only in testing...methods (and tests) are removed